site/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask.ipynb

```diff
@@ -0,0 +1,2084 @@
+       "Tue Dec 27 13:38:40 2016: ncks -3 rasm.nc rasm.nc\n",
+       "history deleted for brevity</dd></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.Dataset>\n",
+       "Dimensions:  (time: 36, x: 275, y: 205)\n",
+       "Coordinates:\n",
+       "    yc       (y, x) float64 dask.array<chunksize=(205, 275), meta=np.ndarray>\n",
+       "    xc       (y, x) float64 dask.array<chunksize=(205, 275), meta=np.ndarray>\n",
+       "  * time     (time) object 1980-09-16 12:00:00 ... 1983-08-17 00:00:00\n",
+       "Dimensions without coordinates: x, y\n",
+       "Data variables:\n",
+       "    Tair     (time, y, x) float64 dask.array<chunksize=(12, 205, 275), meta=np.ndarray>\n",
+       "Attributes:\n",
+       "    title:                     /workspace/jhamman/processed/R1002RBRxaaa01a/l...\n",
+       "    institution:               U.W.\n",
+       "    source:                    RACM R1002RBRxaaa01a\n",
+       "    output_frequency:          daily\n",
+       "    output_mode:               averaged\n",
+       "    convention:                CF-1.4\n",
+       "    references:                Based on the initial model of Liang et al., 19...\n",
+       "    comment:                   Output from the Variable Infiltration Capacity...\n",
+       "    nco_openmp_thread_number:  1\n",
+       "    NCO:                       \"4.6.0\"\n",
+       "    history:                   Tue Dec 27 14:15:22 2016: ncatted -a dimension..."
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_ds = xr.open_mfdataset(paths, combine='by_coords')\n",
+    "new_ds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There you have it 🎉🎉🎉🎉"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This article was inspired by a problem that my colleague was running into. I thought It would be useful to document this fix in a blog post. Many thanks to [Deepak Cherian](https://github.com/dcherian) for pointing me to some relevant xarray issues , [@NCAR/xdev](https://github.com/orgs/NCAR/teams/xdev) team for their valuable and critical feedback on drafts of this essay along the way."
```

Comment:
```suggestion
    "This article was inspired by a problem that my colleague was running into. I thought It would be useful to document this fix in a blog post. Many thanks to [Deepak Cherian](https://github.com/dcherian) for pointing me to some relevant xarray issues and to the [@NCAR/xdev](https://github.com/orgs/NCAR/teams/xdev) team for their valuable and critical feedback on drafts of this essay along the way."
```